Content.Client/EscapeMenu/UI/Tabs/KeyRebindTab.xaml.cs

```diff
@@ -449,6 +477,11 @@ public void UpdateText()
             {
                 Button.Text = Binding?.GetKeyString() ?? Loc.GetString("ui-options-unbound");
             }
+
+            ~BindButton()
+            {
+                configManager.UnsubValueChanged(CVars.DisplayUSQWERTYHotkeys, x => UpdateText());
+            }
```

Comment:
1. You should listen to an event in `InputManager` for mode switch instead, not specifically value changed.
2. Finalizers do not work like this. Subscribe on `EnteredTree` and unsubscribe in `ExitedTree`.
3. Same thing as on the engine-side PR: you're not unsubscribing this delegate correctly because it's a lambda. You need to make a real method to subscribe/unsubscribe for it to actually work.